java/src/jmri/jmrix/mqtt/MqttLightManager.java

```diff
@@ -26,9 +26,9 @@ public void setRcvTopicPrefix(@Nonnull String rcvTopicPrefix) {
     }
     
     @Nonnull
-    public String sendTopicPrefix = "yard/light/";
+    public String sendTopicPrefix = "track/light/";
     @Nonnull
-    public String rcvTopicPrefix  = "yard/light/";
+    public String rcvTopicPrefix  = "track/light/";
```

Comment:
Is this change from "yard/light" to "track/light" something that will require user changes? If so, perhaps it should be mentioned in the release note.

Reply:
There was an inconsistency between the Java file and the bundle file, so I picked the bundle file because that is what is shown in the GUI for a new connection.